trailbot_interfaces/README.md

```diff
@@ -0,0 +1,24 @@
+# trailbot_interfaces
+This package will have contain custom Trailbot services and msgs.
+
+## Make include src 
+```bash
+cd ~/ros2_ws/src/TRAILBot/trailbot_interfaces
+mkdir include src msg
```

Comment:
What is this meant to do?

Reply:
'include' and 'src' folders is what the package creates automatically, so I initially thought building the pkg requires these folders. I tried building the pkg without them and it builds so removing this line.

Reply:
I removed this line